### ISSUE TITLE
Migration of the website

### ISSUE THREAD

**Issue by galexandre** [LiffeyValleyAC/LiffeyValleyAC.github.io#281]
Hi all,
During the last annual meeting, I raised a "problem" with the website. It is difficult to maintain and not very user-friendly. The website is referenced on all our social media and is, a kind of, facade of the club online. 

As a suggestion, I proposed, to migrate the website to a simpler system/platform. My initial idea, is to migrate it to WordPress. 
With WordPress, we have multiple options, we can host it on our own infrastructure (AWS?) or use the services they provide. At the end of the day, it will cost a bit of money.

Do you have any other ideas, platforms that should be considered?

G.

**Comment by Stevecoughlan123** [comment]
Hi Geoffrey,

I certainly agree the website should be up to date and consistent with our social media channels (insta and fb mostly). Keeping all channels in sync has been a challenge for years. I’ve been the FB Admin for ten years, and my own view is that we should now be leading with Insta and syncing with FB and website where appropriate. Certainly our recruitment campaign via Insta during lockdown brought in lots of new members (more than other channels had). 

I know Mike Griffin and Brendan Beere put much work into the Github setup. Check with them before any migration to Wordpress or other platform. Matt Blyth’s view will also be valuable. Thanks too for your own work in this space, it’s often invisible and under-appreciated !

Give me a shout if you’d like to talk - 0863166677

Cheers

Steve

**Comment by mgriffin** [comment]
👋 @galexandre 

I should really stay out of this conversation for two reasons:

1. I built the site so I'm very comfortable with it
2. I'm not even a member of the club anymore 😄 

However...

I'd love to hear what you see that makes the website difficult, is it the site itself or the backend for adding posts to it?

I'm guessing that you've used wordpress before as you bring it up, and it can be a good battle hardened system. I'm not as knowledgeable with it but when I used it in the past, it did require a fair bit of maintenance and tender care to keep it running nicely. This could well be different now, I don't know.

As a little bit of background, myself and Bren did set up this version of the site after a fair bit of back and forth and in the end I decided that a static site was the best option to go with at the time. It was faster than the other options at the time and also completely free to host. While I'm comfortable with it and think it works decently, I'm aware that I'm not in the loop for what the club need now, so I'm happy to hand over anything to anyone that wants to take it over.

**Comment by paulbalfe** [comment]
Hi Mike, 
Thanks for the other TBM changes on the home page which is what we needed.

I’ll come back to you re the difficulties with maintaining the website, they’re mostly training issues from my side.

Regards 
Paul

**Comment by galexandre** [comment]
@Stevecoughlan123 @mgriffin 

I would like to thank, both of you, for your messages. This give me a bit more of context.

Here, is a bit of my thought.

Personally, I don't find it difficult to add anything on the website. Moreover, it works perfectly and, as you mentioned, it is using GitHub page which allow us to host it for free. I had a bit of trouble with the version of Ruby but nothing too difficult. 

However, I work in IT, facing and resolving problems is my day-to-day job, but what about non-IT people? Some persons might find that difficult, as this is not a very user-friendly way to add content (no UI!). Moreover, you have to know Git to add content. Even in IT, some people don't know what it is (true story). This is an obstacle for the creation of content for the website. 

New content would show a certain activity of the club, social media are great for this purpose too. However, the time spend on a post is very short. Showing a sort of activity on the website, show a certain dynamic of the club, which is important for the recruitment of new members (if you put Liffey Valley AC in google, the website is the first result). 
This is why, I came up with the idea of migrating the website to a more user-friendly platform as it will remove some bottlenecks (UI and knowledge of Git). Now, I mentioned WordPress because I gave it a go at some stage in the past, and it seems, according to my recent try, it changed a bit (addition of complexity, slow to design). Some other alternatives exist too. 

In conclusion, the aim that I have in mind is to increase the creation of content for the website. Which could result in a better attractivity for the Club. We can stay on GitHub, this is not a problem, but we would need to have more admins and probably support the creation of content in some ways (tool for the conversion from text to Markdown?).

**Comment by galexandre** [comment]
@mgriffin We were talking today about the website, and we were wondering if you could add Paul, Mat and myself as admin, please?
